usr/share/gamescope-session-plus/modelines/LEN/Go Display.txt

```diff
@@ -0,0 +1,150 @@
+# Legion Go display panel
```

Comment:
Unfortunately, no. `LEN` is the display manufacturer and `Go Display` is the display name. This is how they are identified in my gamescope patch.

But you are right, it would be cleaner and better. In that case, there would need a compile step for quirks that does the renaming at the end though. Or a different mechanism for loading the files.